ui/src/app/shared/type/widget.ts

```diff
@@ -15,6 +15,7 @@ export enum WidgetNature {
     'io.openems.edge.evcs.api.Evcs',
     'io.openems.impl.controller.channelthreshold.ChannelThresholdController', // TODO deprecated
     'io.openems.edge.io.api.DigitalInput',
+    MANAGED_SYMMETRIC_PVINVERTER = 'io.openems.edge.pvinverter.api.ManagedSymmetricPvInverter'
```

Comment:
Are you sure this is a good idea, to add it here? `WidgetNature` is used in `widget.ts` to evaluate the UI widgets. It is not required to add PV-Inverters there.